data/db-schema/src/main/resources/net/corda/db/schema/config/migration/cpx-creation-v1.0.xml

```diff
@@ -209,56 +156,45 @@
             <column name="cpk_file_checksum" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <!-- Check if we could rename this to main_bundle_name -->
-            <column name="bundle_symbolic_name" type="VARCHAR(255)">
+            <column name="cpk_file_name" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <!-- Check if we could rename this to main_bundle_version -->
-            <column name="bundle_version" type="VARCHAR(255)">
+            <column name="cpk_name" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="min_platform_version" type="INT">
+            <column name="cpk_version" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="target_platform_version" type="INT">
+            <column name="cpk_signer_summary_hash" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
```

Comment:
I think `cpk_name`, `cpk_version` and `cpk_signer_summary_hash` should be moved in `cpk_metadata` table, as if kept here it means same CPK under in different CPI (potentially with different filename) can also have different these 3 properties we should not be true?